packages/theme/src/cli/utilities/asset-file-syncer.ts

```diff
@@ -1,192 +1,23 @@
-import {uploadTheme} from './theme-uploader.js'
+import {pollThemeEditorChanges} from './theme-environment/theme-polling.js'
+import {reconcileThemeFiles} from './theme-environment/theme-reconciliation.js'
 import {outputDebug} from '@shopify/cli-kit/node/output'
 import {AdminSession} from '@shopify/cli-kit/node/session'
-import {Checksum, Theme, ThemeAsset, ThemeFileSystem} from '@shopify/cli-kit/node/themes/types'
-import {renderInfo, renderSelectPrompt} from '@shopify/cli-kit/node/ui'
-import {deleteThemeAsset, fetchThemeAsset} from '@shopify/cli-kit/node/themes/api'
+import {Checksum, Theme, ThemeFileSystem} from '@shopify/cli-kit/node/themes/types'
+import {fetchChecksums} from '@shopify/cli-kit/node/themes/api'
 
 export const LOCAL_STRATEGY = 'local'
 export const REMOTE_STRATEGY = 'remote'
 
-type ReconciliationStrategy = typeof LOCAL_STRATEGY | typeof REMOTE_STRATEGY | undefined
-
-interface FilePartitions {
-  localFilesToDelete: Checksum[]
-  filesToDownload: Checksum[]
-  filesToUpload: Checksum[]
```

Comment:
this is no longer going to be partitioned because we are uploading all files in `theme-environment`

Reply:
should I add a comment about this somewhere? Or maybe this file should be part of `theme-reconciliation`....

thoughts?